packages/commons/package.json

```diff
@@ -40,7 +40,8 @@
     "uuid": "9.0.1",
     "winston": "3.13.0",
     "zod": "3.23.8",
-    "zod-validation-error": "3.3.0"
+    "zod-validation-error": "3.3.0",
+    "nodemailer": "^6.9.13"
```

Comment:
Remove `^` so that dependency is pinned (after Renovate setup, we have only pinned dependencies)